utils.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
-var utils = {
+/**
+ * Uses abstract comparison (==) NOT strict comparison (===)
+ */
+function contains(list, item) {
```

Comment:
can we do it through webpack? I assume if you webpack it and _ is redefined
it wont be a problem

On Wed, Aug 10, 2016 at 5:32 PM, Filippo De Santis <notifications@github.com
> wrote:

> In utils.js <https://github.com/p16/fdpn/pull/1#discussion_r74243885>:
>
> > @@ -1,11 +1,147 @@
> >  'use strict';
> >
> > -var utils = {
> > +/**
> > + * Uses abstract comparison (==) NOT strict comparison (===)
> > + */
> > +function contains(list, item) {
>
> @odino <https://github.com/odino>, @shidhinrc and I spoke and I'm going
> to follow these guidelines:
>
>    - we will use lodash
>    - whoever uses this lib would include lodash otherwise things are
>    gonna break :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/p16/fdpn/pull/1/files/7fc4dc404e388c1983ccbf594509469bfae1cbbc#r74243885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAUC5IWVeMaekG6DuJ7aD7lcuxIQC9jJks5qedLcgaJpZM4JgR6V>
> .
>



-- 
Nadalin Alessandro
http://www.linkedin.com/in/alessandronadalin
www.odino.org
www.twitter.com/_odino_